frontend/src/pages/projects/screens/projects/ProjectListView.tsx

```diff
@@ -1,41 +1,73 @@
 import * as React from 'react';
 import { useNavigate } from 'react-router-dom';
 import { Table } from '#~/components/table';
-import { ProjectKind } from '#~/k8sTypes';
+import { KnownLabels, ProjectKind } from '#~/k8sTypes';
 import { getProjectOwner } from '#~/concepts/projects/utils';
 import { ProjectsContext } from '#~/concepts/projects/ProjectsContext';
 import ProjectTableRow from '#~/pages/projects/screens/projects/ProjectTableRow';
 import { getDisplayNameFromK8sResource } from '#~/concepts/k8s/utils';
 import DashboardEmptyTableView from '#~/concepts/dashboard/DashboardEmptyTableView';
 import ProjectsToolbar from '#~/pages/projects/screens/projects/ProjectsToolbar';
 import {
+  aiProjectFilterKey,
   initialProjectsFilterData,
   ProjectsFilterDataType,
 } from '#~/pages/projects/screens/projects/const';
 import { SupportedArea, useIsAreaAvailable } from '#~/concepts/areas';
+import { useBrowserStorage } from '#~/components/browserStorage/BrowserStorageContext';
 import { columns } from './tableData';
 import DeleteProjectModal from './DeleteProjectModal';
 import ManageProjectModal from './ManageProjectModal';
 
+const PROJECT_FILTER_STORAGE_KEY = 'odh.dashboard.projects.type.filter';
+
 type ProjectListViewProps = {
   allowCreate: boolean;
 };
 
+export const isAiProject = (project: ProjectKind): boolean => {
+  return project.metadata.labels?.[KnownLabels.DASHBOARD_RESOURCE] === 'true';
+};
+
+const getAiProjects = (projects: ProjectKind[]) => {
+  return projects.filter((project) => {
+    return isAiProject(project);
+  });
+};
+
 const ProjectListView: React.FC<ProjectListViewProps> = ({ allowCreate }) => {
   const { projects } = React.useContext(ProjectsContext);
   const navigate = useNavigate();
+  const [projectFilter, setProjectFilter] = useBrowserStorage<string>(
+    PROJECT_FILTER_STORAGE_KEY,
+    aiProjectFilterKey,
+    true,
+    true,
+  );
+
   const [filterData, setFilterData] =
     React.useState<ProjectsFilterDataType>(initialProjectsFilterData);
-  const onClearFilters = React.useCallback(
-    () => setFilterData(initialProjectsFilterData),
-    [setFilterData],
-  );
+
+  const [aiProjectNum, setAiProjectNum] = React.useState(0);
+  const [fullProjectNum, setFullProjectNum] = React.useState(projects.length || 0);
+
+  React.useEffect(() => {
+    setAiProjectNum(getAiProjects(projects).length || 0);
+    setFullProjectNum(projects.length || 0);
+  }, [projects]);
+
   const filteredProjects = React.useMemo(
     () =>
       projects.filter((project) => {
         const nameFilter = filterData.Name?.toLowerCase();
         const userFilter = filterData.User?.toLowerCase();
+        const aiProjectFilter = projectFilter === aiProjectFilterKey;
 
+        if (aiProjectFilter) {
+          if (!isAiProject(project)) {
+            return false;
+          }
+        }
```

Comment:
nit: combine if statements.

Reply:
@christianvogt  i made all the changes locally and tested them successfully; but git is being weird so not able to push it at this time.

Reply:
it's a github thing; apparently: https://www.githubstatus.com/incidents/5q7nmlxz30sk